apps/fl/download.html

```diff
@@ -29,7 +29,6 @@
       <form role="dialog" data-type="confirm">
         <section>
           <h1 data-l10n-id="download-progress-title"></h1>
-          <p style="text-align: center;">
             <progress id="direct-download-progress" class="pack-activity" value="0"></progress>
           </p>
```

Comment:
That comment was supposed to read: you deleted the &lt;p&gt; tag bug forgot to delete the &lt;/p&gt; tag.